app/views/product_users/index.html.haml

```diff
@@ -20,6 +20,12 @@
   %p.notice= text("none")
 - else
   - if @product.has_product_access_groups?
+    = form_tag facility_instrument_users_path(format: :csv), method: :get, class: "search_form" do
+      = hidden_field_tag :email, current_user.email, disabled: true
```

Comment:
Having a hidden field like this is a smell. Can we instead use `current_user.email` in the receiving controller method?

Reply:
It looks like `CsvEmailAction` defaults to `current_user.email`, and we don't need to prompt the user for an email address, so I think we can drop the hidden fields.